### ISSUE TITLE
Flipping webcam image

### ISSUE THREAD

**Issue by cass00** [umlaeute/v4l2loopback#106]
I'm trying to rotate the output of my webcam using v4l2loopback via gst-launch.

The following command works:

```
gst-launch-1.0 -v v4l2src  ! 'video/x-raw,width=640,format=YUY2' ! videoconvert ! videoflip method=clockwise ! xvimagesink
```

However if I'm trying to pipe into a v4l2sink, i.e.

```
gst-launch-1.0 -v v4l2src  ! 'video/x-raw,width=640,format=YUY2' ! videoconvert ! videoflip method=clockwise ! v4l2sink device=/dev/video2
```

I'm getting:

```
ERROR: from element /GstPipeline:pipeline0/GstV4l2Src:v4l2src0: Internal data flow error.
Additional debug info:
gstbasesrc.c(2943): gst_base_src_loop (): /GstPipeline:pipeline0/GstV4l2Src:v4l2src0:
streaming task paused, reason error (-5)
Execution ended after 0:00:00.313524448
```

Without the flipping the above command works.

**Comment by umlaeute** [comment]
what exactly do you mean with *Without the flipping the above command works.*?

**Comment by cass00** [comment]
```gst-launch-1.0 -v v4l2src  ! 'video/x-raw,width=640,format=YUY2' ! videoconvert ! v4l2sink device=/dev/video2```

works

**Comment by umlaeute** [comment]
hopefully fixed in `0.10.0`

**Comment by cass00** [comment]
Can confirm that it works now.
